src/styles/calendar/calendar.scss

```
@@ -0,0 +1,48 @@
+@use '../utils/variables' as *;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  width: $calendar-width;
+  gap: $gap;
+  align-content: flex-end;
+
+  @for $i from 1 through 31 {
+    &--month-length-#{$i} div:nth-child(-n + #{$i}) {
```

[checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css): Don't use tag names for styling (except html and body)
```suggestion
    &--month-length-#{$i} &__day:nth-child(-n + #{$i}) {
```